commercetools-models/src/test/java/io/sphere/sdk/subscriptions/SubscriptionFixtures.java

```diff
@@ -118,7 +149,7 @@ public static Subscription createSubscription(final BlockingSphereClient client,
 
     public static void deleteSubscription(final BlockingSphereClient client, final Subscription subscription) {
         if (subscription != null) {
-            client.executeBlocking(SubscriptionDeleteCommand.of(subscription));
+            Subscription subs = client.executeBlocking(SubscriptionDeleteCommand.of(subscription));
```

Comment:
Since we don't use the `subs` return value here, I would suggest to just call `client.executeBlocking` without storing the return value.